ipaserver/install/installutils.py

```diff
@@ -1044,18 +1044,24 @@ def in_container():
        systemd-detect-virt requires the whole systemd subsystem which
        isn't a reasonable require in a container.
     """
+    data_sched = ""
```

Comment:
I suppose it's a few nsec faster to not assign and split an empty list. I think I'd prefer assigning None instead of an empty string though to data_sched and data_cgroup.

If you rebase to current HEAD you'll pick up the fix for the failing tests.